datasets/snli/README.md

```diff
@@ -1,4 +1,37 @@
+---
+language: 
+- en
+task:
+- text-classification
+purpose:
+- NLI
+size:
+- ">100k"
```

Comment:
Maybe not in this particular PR's scope, but do we already have a sense of the taxonomy i.e. the possible values for those tags? (to display them in the website)

Reply:
@mcmillanmajora is working on that next (after taking a look at more of our datasets to get of sense of what we need)